src/app/extract_blocks/extract_blocks.ml

```diff
@@ -1,4 +1,5 @@
 (* extract_blocks.ml -- dump extensional blocks from archive db *)
+[@@@coverage exclude_file]
```

Comment:
This change is for disabling coverage for this file as it has compilation issue when building with `--instrument-with bisect_ppx`